### DISCUSSION TITLE
Window moving outlines

### DISCUSSION THREAD

**Discussion by twylightproto** [valinet/ExplorerPatcher#3594] [Q&A]
Excuse me if this is just me being a dumbass, but I'm trying to figure out how to change from the way the windows move now (an inverted outline) to making them move real-time. The way it works now kind of... makes me lose focus, if that makes sense? I don't know, it just bugs me a lot. Just wanted to ask if this was reversable. Thanks!

**Comment by pyrates999** [comment]
https://www.youtube.com/watch?v=nW9ayOY6tbY

**Reply by twylightproto** [reply]
Thanks! I didn't know about that menu.
